app/uk/gov/hmrc/gform/sharedmodel/form/FormIdData.scala

```diff
@@ -44,6 +44,11 @@ sealed trait FormIdData {
     fold[FormIdData](_.copy(formTemplateId = formTemplate.originalId)) {
       _.copy(formTemplateId = formTemplate.originalId)
     }
+
+  def withTemplateId(formTemplateID: FormTemplateId): FormIdData =
```

Comment:
Please rename `formTemplateID` to `formTemplateId` to be consistent with the rest of the code base